README.md

```diff
@@ -12,11 +12,8 @@
 This project uses <https://docusaurus.io/> under the hood which is a documentation generator comprised of React and Markdown.
 
 1. Git clone this repository
-
-2. `yarn` install dependencies in the root directory
-
-3.`cd website` switch to the website directory
-
+3. `cd website` switch to the website directory
```

Comment:
Thank you! I didn't noticed this. I'm used to the `package.json` being in the root.